packages/lodestar/src/metrics/validatorMonitor.ts

```diff
@@ -204,8 +212,22 @@ export function createValidatorMonitor(
         } else {
           metrics.validatorMonitor.prevEpochOnChainTargetAttesterMiss.inc({index});
         }
-        if (summary.inclusionDistance) {
-          metrics.validatorMonitor.prevEpochOnChainInclusionDistance.set({index}, summary.inclusionDistance);
+        const prevEpochSummary = monitoredValidator.summaries.get(previousEpoch);
+        const attestationCorrectHead = prevEpochSummary?.attestationCorrectHead;
+        if (attestationCorrectHead !== null && attestationCorrectHead !== undefined) {
+          metrics.validatorMonitor.prevOnChainAttesterCorrectHead.set({index}, attestationCorrectHead ? 1 : 0);
+        }
+        const attestationMinBlockInclusionDistance = prevEpochSummary?.attestationMinBlockInclusionDistance;
+        let inclusionDistance;
+        if (attestationMinBlockInclusionDistance != null && attestationMinBlockInclusionDistance > 0) {
+          // altair, attestation is not missed
+          inclusionDistance = attestationMinBlockInclusionDistance;
+        } else if (summary.inclusionDistance) {
+          // phase0, this is from the state transition
+          inclusionDistance = summary.inclusionDistance;
+        }
+        if (inclusionDistance !== undefined) {
+          metrics.validatorMonitor.prevEpochOnChainInclusionDistance.set({index}, inclusionDistance);
         }
```

Comment:
Please add white space to this blocks, for example
- prepare variables
- large ternary logic block
- commit variables

```
const prevEpochSummary = monitoredValidator.summaries.get(previousEpoch);
const attestationCorrectHead = prevEpochSummary?.attestationCorrectHead;
if (attestationCorrectHead !== null && attestationCorrectHead !== undefined) {
  metrics.validatorMonitor.prevOnChainAttesterCorrectHead.set({index}, attestationCorrectHead ? 1 : 0);
}

const attestationMinBlockInclusionDistance = prevEpochSummary?.attestationMinBlockInclusionDistance;
const inclusionDistance =
  attestationMinBlockInclusionDistance != null && attestationMinBlockInclusionDistance > 0
    ? // altair, attestation is not missed
      attestationMinBlockInclusionDistance
    : summary.inclusionDistance
    ? // phase0, this is from the state transition
      summary.inclusionDistance
    : null;

if (inclusionDistance !== null) {
  metrics.validatorMonitor.prevEpochOnChainInclusionDistance.set({index}, inclusionDistance);
}
```